joomla/joomla-cms
# [3.8] Classmap deprecated class stubs

### Summary of Changes

Class map the deprecated `JWeb` and `JWebClient` class stubs to their non-deprecated namespaced counterparts.

### Testing Instructions

Use of these class names should still work without change.  Similar to every other namespacing PR.
